aslam_cv_matcher/src/gyro-two-frame-matcher.cc

```diff
@@ -1,6 +1,13 @@
 #include "aslam/matcher/gyro-two-frame-matcher.h"
 
 #include <aslam/common/statistics/statistics.h>
+#include <glog/logging.h>
+
+DEFINE_int32(gyro_matcher_small_search_distance_px, 10, 
+    "Small image space distances for keypoint matches.");
```

Comment:
What do "image space distances" actually mean? :-)

Reply:
I wrote that some time ago in a hurry. Feel free to rephrase :)

Reply:
Ok :-)